packages/common/mui/src/lib/card-variants/currency-fixed/currency-fixed.tsx

```diff
@@ -0,0 +1,190 @@
+import { useState, ReactNode } from 'react';
+import { styled, Box } from '@mui/material';
+import { TokenIcon } from '@notional-finance/icons';
+import { Button } from '../../button/button';
+import { Card } from '../../card/card';
+import { Link } from 'react-router-dom';
+import {
+  CurrencyTitle,
+  H4,
+  SectionTitle,
+  CardInput,
+  Subtitle,
+} from '../../typography/typography';
+import { Market } from '@notional-finance/sdk/src/system';
+import { FormattedMessage } from 'react-intl';
+import { useNotionalTheme, NotionalTheme } from '@notional-finance/styles';
+
+export interface CurrencyFixedProps {
+  rate: number;
+  allRates: Record<number, number>;
+  symbol?: string;
+  route: string;
+  buttonText: ReactNode;
+}
+
+export interface ContentWrapperProps {
+  hovered: boolean;
+  theme: NotionalTheme;
+}
+
+export const CurrencyFixed = (props: CurrencyFixedProps) => {
+  const theme = useNotionalTheme('light');
+  const { route, symbol, buttonText, rate, allRates } = props;
+  const [hovered, setHovered] = useState(false);
+  const formattedRate = Market.formatInterestRate(rate, 2);
+
+  return (
+    <Link to={route}>
+      <Card
+        height={'auto'}
+        onMouseOver={() => setHovered(true)}
+        onMouseOut={() => setHovered(false)}
+      >
+        {symbol && (
+          <StyledIcon top={theme.spacing(-9)}>
+            <TokenIcon symbol={symbol} size="extraLarge" />
+          </StyledIcon>
+        )}
+        <Box sx={{ overflow: 'hidden' }}>
+          <CurrencyTitle
+            accent
+            textAlign="left"
+            marginBottom={theme.spacing(2)}
+          >
+            {symbol}
+          </CurrencyTitle>
+          <ContentWrapper hovered={hovered} theme={theme}>
+            <Box>
+              {allRates[0] && (
```

Comment:
Return the formatted date instead of 3 month, 6 month 1 year. The date format should be `Abbr Month Year`